src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -855,6 +833,56 @@ fn insert_link_into_header(
     )
 }
 
+// Convert fontawesome `<i>` tags to inline SVG
+fn convert_fontawesome(html: &str) -> String {
+    use font_awesome_as_a_crate as fa;
+
+    let regex = Regex::new(r##"<i([^>]+)class="([^"]+)"([^>]*)></i>"##).unwrap();
+    regex
+        .replace_all(html, |caps: &Captures<'_>| {
+            let text = &caps[0];
+            let before = &caps[1];
+            let classes = &caps[2];
+            let after = &caps[3];
+
+            let mut icon = String::new();
+            let mut type_ = fa::Type::Regular;
+            let mut other_classes = String::new();
+
+            for class in classes.split(" ") {
+                if let Some((prefix, remainder)) = class.split_once('-') {
+                    if let Some(t) = match prefix {
+                        "fa" => Some(fa::Type::Regular),
+                        "fas" => Some(fa::Type::Solid),
+                        "fab" => Some(fa::Type::Brands),
+                        _ => None,
+                    } {
+                        type_ = t;
+                        icon = remainder.to_owned();
+                        continue;
+                    }
+                }
+                other_classes += " ";
+                other_classes += class;
+            }
+
+            if icon.is_empty() {
+                text.to_owned()
+            } else if let Ok(svg) = fa::svg(type_, &icon) {
```

Comment:
I wonder if, because older versions of fontawesome didn't have the "brands" category, we should try to use a fallback to cover old books. For example:

```suggestion
            } else if let Ok(svg) = fa::svg(type_, &icon).or_else(|| fa::svg(fa::Type::Brands, &icon)) {
```